docs/pages/contributing/documentation/terms-usage.mdx

```diff
@@ -0,0 +1,171 @@
+## Terminology and usage conventions
+
+This section covers common terminology, style, and usage questions.
+It includes recommended practices for using punctuation, referring to user interface
+elements, and selecting the right words for the information you're trying to convey. 
+
+### Above and below
+
+Don't use *above* to mean *earlier* or as an adjective preceding a noun (*the above section*) 
+or following a noun (*the code above*). Use a hyperlink, or use *previous*, *preceding*, or 
+*earlier*. Don't use *below* to mean *later* or as an adjective preceding a noun (*the below 
+section*) or following a noun (*the code below*). Use a hyperlink, or use *later* or 
+*the following*. For example: 
+
+- Use the preceding code to display information about the database.
+- Use the following code to display information about the database.
+
+### Dates and numbers
+
+Date formats can depend on context, including the location of a computer and a user's 
+preferences. In most cases, you can use the DD Mon YYYY or DD Month YYYY format for dates. 
+In body text, spell out whole numbers from zero through nine. Use numerals for 10 or greater. 
+Use commas in numbers with four or more digits. Use *more than* instead of *over* 
+(over is a spatial term). 
+
+### Emphasis and special notices
+
+Use bold formatting for user interface elements that the user interacts with, including:
+
+- Menus and menu commands.
+- Field and button labels.
+- Options that can be selected in the user interface.
+
+If there's text that requires more attention than the surrounding body, consider isolating 
```

Comment:
I would separate this H3 into H4s. I was reading this paragraph as an extension of the discussion of UI element formatting and got confused for a bit.